WooCommerce/Classes/ViewRelated/Orders/Cells/OrderTableViewCell.swift

```diff
@@ -39,7 +39,7 @@ final class OrderTableViewCell: UITableViewCell & SearchResultCell {
     /// Renders the specified Order ViewModel
     ///
     func configureCell(viewModel: OrderDetailsViewModel, orderStatus: OrderStatus?) {
-        titleLabel.text = viewModel.summaryTitle
+        titleLabel.text = title(for: viewModel.order)
```

Comment:
as you mentioned in the PR description, that's interesting this order list cell also uses `OrderDetailsViewModel` 🤔 it probably makes more sense to have a lighter weight view model for the order list cell

Reply:
thanks for creating the issue!